client/src/style/scss/overrides.scss

```diff
@@ -88,7 +88,7 @@ button {
 // Modal -- wider by default, scroll like Trello
 
 .modal-dialog {
-    width: 690px;
+    // width: 690px; // by removing this, size can be set with the "size" attribute of the bootstrap component <b-modal>
```

Comment:
:warning: this could have unforeseen consequences

Reply:
I think this is OK for now, we can see if somebody has any concerns when this goes upstream :)